cinch/playbooks/jenkins_backup.yml

```diff
@@ -29,7 +29,7 @@
       diff_schedule: H 0 * * *
       max_sets: 2
       exclude: ''
-      wait_for_idle: true
+      wait_for_idle: false
       quiet_mode_timeout: 480
```

Comment:
This has no effect with `waitForIdle` turned off. You might want to delete it or comment it in case we will ever bring this back.

Reply:
Probably https://github.com/RedHatQE/cinch/pull/150/files#diff-75c7e2bf9ddde36511712d435e5cf082R53 should be triggered conditionally iff `waitForIdle` is true too.